src/main/java/core/basesyntax/strategy/handler/BalanceHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void apply(Fruit fruit, int quantity) {
```

Comment:
The method `apply` should not begin with an empty line according to the checklist.